### PR TITLE
Version 0.20.0

### PR DESCRIPTION
**Pull request by Kludex**
GitHub Release Draft: [Version 0.20.0 Draft](https://github.com/encode/starlette/releases/tag/untagged-d4f5bcbb1ea9702f0555)
Commit diff between last tag and master: https://github.com/encode/starlette/compare/0.19.1...master

Checklist:
- [x] https://github.com/encode/starlette/pull/1590
- [x] https://github.com/encode/starlette/pull/1525
- ~https://github.com/encode/starlette/pull/1449~ moved to the next milestone
- ~https://github.com/encode/starlette/pull/1377~ moved to the next milestone

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 0.20.0
+
+April 22, 2022
```
Comment (Kludex) [inline docs/release-notes.md]:
```suggestion
April 28, 2022
```

File: docs/release-notes.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## 0.20.0
+
+April 28, 2022
+
+### Removed
+* Drop Python 3.6 support [#1357](https://github.com/encode/starlette/pull/1357).
```
Comment (aminalaee) [inline docs/release-notes.md]: Shouldn't the checklist be added here?

Comment (Kludex) [reply docs/release-notes.md]: What items do you think should be included?

Comment (Kludex) [reply docs/release-notes.md]: We don't have written rules for the changelog on starlette, but tests, docs and type hints modifications were never included.

Comment (aminalaee) [reply docs/release-notes.md]: Yeah I meant the doc changes, but you're right, they were never part of release changes.

File: docs/release-notes.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
+## 0.20.0
+
+April 28, 2022
```
Comment (aminalaee) [inline docs/release-notes.md]: Friendly reminder about this :)

**Review by aminalaee — APPROVED** [review]

File: docs/release-notes.md (hunk 3 above)
Comment (Kludex) [reply docs/release-notes.md]: Thanks! I would have forgotten! 😅

Comment (Kludex) [inline docs/release-notes.md]:
```suggestion
May 3, 2022
```

**Pull request merged**